go.mod

```diff
@@ -27,17 +27,19 @@ require (
 	knative.dev/eventing v0.32.0
 	knative.dev/eventing-kafka v0.32.0
 	knative.dev/eventing-kafka-broker v0.0.0-00010101000000-000000000000
-	knative.dev/hack v0.0.0-20220524153203-12d3e2a7addc
+	knative.dev/hack v0.0.0-20220601014117-bafe4b15ca50
 	knative.dev/networking v0.0.0-20220412163509-1145ec58c8be
 	knative.dev/operator v0.31.2
-	knative.dev/pkg v0.0.0-20220524202603-19adf798efb8
-	knative.dev/serving v0.30.0
+	knative.dev/pkg v0.0.0-20220601013938-bac16f2394ce
+	knative.dev/serving v0.32.0
```

Comment:
The bump of serving, does it need to be in here? Or can it be separated? E.g. eventing seems to be already on 0.32 :man_shrugging:

Reply:
Yes I tried to align those. v0.32.0 is Serving 1.5 afaik but for operator is not. I could replace it to keep it as is.

Reply:
Actually no, we already replace the versions with the midstream ones so we are ok.